securedrop/journalist_app/main.py

```diff
@@ -42,16 +42,14 @@ def login() -> Union[str, werkzeug.Response]:
                 db.session.commit()
 
                 session['uid'] = user.id
-                session['nonce'] = user.session_nonce
+                session.regenerate()
                 return redirect(url_for('main.index'))
 
         return render_template("login.html")
 
     @view.route('/logout')
     def logout() -> werkzeug.Response:
-        session.pop('uid', None)
-        session.pop('expires', None)
-        session.pop('nonce', None)
+        flash(session.destroy())
```

Comment:
Clearly flash here was left when debugging and should just be `session.destroy()`.